### ISSUE TITLE
nddata leak test failure

### ISSUE THREAD

**Issue by MSeifert04** [astropy/astropy#6426]
@mhvk reported a failing test `test_leak` ([here](https://github.com/astropy/astropy/pull/6176#issuecomment-320479364)).

@mhvk Do you have a link to the log where it failed? It would be interesting to know what the exception message was.

**Comment by AustereCuriosity** [comment]
Here it is! [Failing Job](https://travis-ci.org/astropy/astropy/jobs/261195682)

**Comment by MSeifert04** [comment]
@AustereCuriosity Thank you 👍

**Comment by MSeifert04** [comment]
That's really weird. I ran this test semi-manually thousands of times and I can't reproduce the failure locally.

**Comment by MridulS** [comment]
The travis report isn't accessible anymore and it seems like it has been reported again in 7 years. I guess it's safe to close this issue?

**Comment by pllim** [comment]
Yeah, let's close. Nothing much in the description to go on from. Thanks for the ping!
